lib/internal/fs/dir.js

```diff
@@ -34,74 +33,73 @@ const {
   validateUint32,
 } = require('internal/validators');
 
-const kDirHandle = Symbol('kDirHandle');
-const kDirPath = Symbol('kDirPath');
-const kDirBufferedEntries = Symbol('kDirBufferedEntries');
-const kDirClosed = Symbol('kDirClosed');
-const kDirOptions = Symbol('kDirOptions');
-const kDirReadImpl = Symbol('kDirReadImpl');
-const kDirReadPromisified = Symbol('kDirReadPromisified');
-const kDirClosePromisified = Symbol('kDirClosePromisified');
-const kDirOperationQueue = Symbol('kDirOperationQueue');
-
 class Dir {
+  #dirHandle;
+  #dirPath;
+  #dirBufferedEntries;
+  #dirClosed;
+  #dirOptions;
+  #dirReadPromisified;
+  #dirClosePromisified;
+  #dirOperationQueue;
+
   constructor(handle, path, options) {
     if (handle == null) throw new ERR_MISSING_ARGS('handle');
-    this[kDirHandle] = handle;
-    this[kDirBufferedEntries] = [];
-    this[kDirPath] = path;
-    this[kDirClosed] = false;
+    this.#dirHandle = handle;
+    this.#dirBufferedEntries = [];
```

Comment:
These default values can be moved to the property declarations.

Reply:
Done. Thank you for the quick review.